docs/src/modules/utils/parseMarkdown.test.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import { getContents } from './parseMarkdown';
+
+describe('parseMarkdown', () => {
```

Comment:
This is the kind of test where I would prefer fixtures + snapshots but it's not worth it to set this up for one test. If this test becomes an issue (breaks, readability etc) we can revisit.